README.md

```diff
@@ -3,7 +3,18 @@
 [![Fosstars security rating](https://raw.githubusercontent.com/SAP/cloud-security-xsuaa-integration/fosstars-report/fosstars_badge.svg)](https://github.com/SAP/cloud-security-xsuaa-integration/blob/fosstars-report/fosstars_report.md)
 [![CodeQL](https://github.com/SAP/cloud-security-xsuaa-integration/workflows/CodeQL/badge.svg)](https://github.com/SAP/cloud-security-xsuaa-integration/actions?query=workflow%3ACodeQL)
 
-
+# :warning: Announcement of End of Support for our Spring Libraries
+As support for [Spring Boot 2.x](https://spring.io/projects/spring-boot#support) and [Spring Framework 5.x](https://spring.io/projects/spring-framework#support) comes to an end, we will also discontinue support for our Spring-dependent artifacts. This means that, beyond the end of support date for affected libraries, we will not provide any bug fixes nor security patches.
+
+We recommend upgrading to our [3.x major release](https://github.com/SAP/cloud-security-xsuaa-integration/tree/main), which supports Spring Boot 3.x, Spring Framework 6.x, and Java 17. For assistance with the migration process, please refer to our [migration guide](./MIGRATION_TO_3.x.md).
+
+| ArtifactId                                  | End of Support |
+|---------------------------------------------|----------------|
+| spring-xsuaa                                | Dec, 2025      |
+| spring-xsuaa-test                           | Dec, 2025      |
+| xsuaa-spring-boot-starter                   | Nov, 2023      |
+| spring-security                             | Dec, 2025      |
```

Comment:
Can we make it a bit clearer that spring-security will continue to be supported in the 3.x branch?
Maybe we could add a column to the table for 'Version' that says 2.x for these artefacts?